examples/kafka-broker/verify.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+export NAME=kafka-broker
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Waiting for Kafka on port 9092"
+wait_for 120 sh -c "docker-compose logs -f kafka | grep -q 'started (kafka.server.KafkaServer)'"
```

Comment:
rather than doing this can you add healthchecks and waits in the docker-compose file

Reply:
I only found this way to make sure Kafka launched successfully. Because it takes some seconds to connect to the ZooKeeper and sometimes it gets failed when starting.

Reply:
Updated, please help verify again